core/src/main/java/org/elasticsearch/index/mapper/internal/SeqNoFieldMapper.java

```diff
@@ -108,6 +114,34 @@ public Query termQuery(Object value, @Nullable QueryShardContext context) {
             throw new QueryShardException(context, "SeqNoField field [" + name() + "] is not searchable");
         }
 
+        @Override
+        public FieldStats stats(IndexReader reader) throws IOException {
```

Comment:
can we add a nocommit on this implementation, so it will be removed? 
PS. Although we do this temporarily to achieve other goals,  we will end up adding a different stats implementation once seq no is properly indexed.